samza-sql/src/main/java/org/apache/samza/sql/udf/ReflectionBasedUdfResolver.java

```diff
@@ -49,20 +54,26 @@
   private static final Logger LOG = LoggerFactory.getLogger(ReflectionBasedUdfResolver.class);
 
   private static final String CONFIG_PACKAGE_PREFIX = "samza.sql.udf.resolver.package.prefix";
```

Comment:
remove this ?

Reply:
Discussed offline. We'd planned to proceed with having both the configurations for now.